golem-worker-bridge/src/api/custom_request_endpoint.rs

```diff
@@ -72,46 +74,55 @@ impl CustomRequestEndpoint {
             req_body: json_request_body,
         };
 
-        let api_definition_id_res = headers
-            .iter()
-            .find(|(key, _)| key.as_str().to_lowercase() == "x-api-definition-id")
-            .map(|(_, value)| value)
-            .ok_or("Missing X-API-Definition-Id header");
-
-        let api_definition_id_header = match api_definition_id_res {
-            Ok(api_definition_id) => api_definition_id,
+        let api_definition_id = match get_header_value(&headers, GOLEM_API_DEFINITION_ID_EXTENSION)
+        {
+            Ok(api_definition_id) => ApiDefinitionId(api_definition_id.to_string()),
             Err(err) => {
+                error!(
+                    "{}",
+                    format!(
+                        "Invalid {} header: {}",
+                        GOLEM_API_DEFINITION_ID_EXTENSION, err
+                    )
+                );
                 return Response::builder()
                     .status(StatusCode::BAD_REQUEST)
                     .body(Body::from_string(err.to_string()));
             }
         };
 
-        let api_definition_id_string = api_definition_id_header
-            .to_str()
-            .map_err(|e| format!("Invalid X-API-Definition-Id header: {}", e));
-
-        let api_definition_id = match api_definition_id_string {
-            Ok(api_definition_id) => ApiDefinitionId(api_definition_id.to_string()),
+        let version = match get_header_value(&headers, GOLEM_API_DEFINITION_VERSION) {
+            Ok(version) => Version(version),
             Err(err) => {
-                error!("Invalid X-API-Definition-Id header: {}", err);
+                error!(
+                    "{}",
+                    format!("Invalid {} header: {}", GOLEM_API_DEFINITION_VERSION, err)
```

Comment:
error in `get_header_value` 
is already formatted like `format!("Invalid value for the header {} error: {}", header_name, e)`